client/my-sites/plans-v2/types.ts

```diff
@@ -60,7 +60,7 @@ export interface SelectorProduct extends SelectorProductCost {
 	description: TranslateResult | ReactNode;
 	term: Duration;
 	buttonLabel?: TranslateResult;
-	features: Features;
+	features: ProductCardFeatures;
```

Comment:
It doesn't feel right to me to constraint/type the `features` of a plan/product to something more low-level such a ProductCard. What if we want to display these features outside of a `JetpackProductCard`? My intuition tells me that this new type `ProductCardFeatures` should live inside of `client/my-sites/plans-v2/types.ts` and anything from `'components/jetpack/card/jetpack-product-card` should import the type from this new location. If we do this, we should probably rename the type to something more generic such as `SelectorProductFeatures` or to something that is not tied to an UI implementation.

Reply:
You're right, I'll do the update.